### PR TITLE
Redirect when found preferred upload endpoint on /content/add

### PR DESCRIPTION
**Pull request by gmelodie**
Adds the functionality requested in #153

### CODE REVIEW

**Review by jimmylee — APPROVED** [review]
LGTM

File: handlers.go [hunk 1]
```diff
@@ -694,19 +695,19 @@ func (s *Server) handleAdd(c echo.Context, u *User) error {
 	defer span.End()
 
 	if s.CM.contentAddingDisabled || u.StorageDisabled || s.CM.localContentAddingDisabled {
-		var details string
 		if s.CM.localContentAddingDisabled {
 			uep, err := s.getPreferredUploadEndpoints(u)
+			uep = []string{"http://localhost:3005/content/add"}
```
Comment (en0ma) [inline handlers.go:700]: you missed removing this @gmelodie

Comment (gmelodie) [reply handlers.go:700]: *shyface*

**Review by en0ma — APPROVED** [review]
LGTM

**Review by en0ma — APPROVED** [review]
LGTM

**Review by whyrusleeping — CHANGES_REQUESTED** [review]
Redirects don't work for uploads unfortunately, this will just result in a broken request.

**Pull request merged**